server/events/project_command_builder.go

```diff
@@ -395,9 +395,9 @@ func (p *DefaultProjectCommandBuilder) buildAllCommandsByCfg(ctx *command.Contex
 		ctx.Log.Info("automatically determined that there were %d projects modified in this pull request: %s", len(modifiedProjects), modifiedProjects)
 		for _, mp := range modifiedProjects {
 			ctx.Log.Debug("determining config for project at dir: %q", mp.Path)
-			pWorkspace, err := p.ProjectFinder.DetermineWorkspaceFromHCL(ctx.Log, repoDir)
+			pWorkspace, err := p.ProjectFinder.DetermineWorkspaceFromHCL(ctx.Log, repoDir+"/"+mp.Path)
```

Comment:
Please write a test condition for this that would have failed if this wasn't added

Reply:
@nitrocode Any suggestions on where to write such a test? I'd image we'd want a test for the caller `buildAllCommandsByCfg` here but I don't see any test cases for the caller itself in the codebase.

Reply:
hm, i'm taking a look at this now; the naïve and perhaps un-useful answer is "in `server/events/project_command_builder_test.go`, that's the conventional place where Go tooling expects to find tests associated with this code.

@mprokopov can you provide an example of an `atlantis.yaml` file that demonstrates this problem?  or an example of a directory structure that demonstrates it?  i don't think i know what "Workspace detection for Terraform Cloud" means, sorry.